space.fips.Fips.json

```diff
@@ -0,0 +1,42 @@
+{
+	"app-id": "space.fips.Fips",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.9",
+	"sdk": "org.kde.Sdk",
+	"command": "fips",
+	"rename-icon": "fips",
+	"rename-desktop-file": "fips.desktop",
+	"finish-args": [
+		"--share=ipc",
+		"--share=network",
+		"--socket=x11",
+		"--socket=wayland",
+		"--filesystem=home:ro",
+		"--device=dri"
+	],
+	"modules": [
+		{
+			"name": "fips",
+			"buildsystem": "cmake",
+			"config-opts": [
+				"-DCMAKE_BUILD_TYPE=RelWithDebugInfo"
+			],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://github.com/matwey/fips3",
```

Comment:
Flathub requires stable and reproducible builds. You should use a specific commit here or a stable tarball.

Reply:
Absolutely agree. But then we will have to wait a little until we tag 3.3.0 release. Until then I would like to continue fixing other issues on Flatpak package.